modin/experimental/core/execution/native/implementations/hdk_on_native/io/io.py

```diff
@@ -227,44 +114,51 @@ def read_csv(
         -----
         Reading performed by using of `pyarrow.read_csv` function.
         """
-        items = locals().copy()
-        mykwargs = {k: items[k] for k in items if k in cls.arg_keys}
-        eng = str(engine).lower().strip()
+        eng = str(kwargs["engine"]).lower().strip()
         try:
             if eng in ["pandas", "c"]:
```

Comment:
```suggestion
            if eng in ("pandas", "c"):
```
let's not make lists when they're not needed - they're slower and take more memory